spec/services/csv/lettings_log_csv_service_spec.rb

```diff
@@ -89,7 +89,7 @@
 
       it "removes some log attributes related to questions from the headers and replaces them with their derived values in the correct order" do
         expect(headers).not_to include "rent_type"
-        expect(headers).to include(*%w[wrent renttype rent_type_detail])
+        expect(headers).to include(*%w[wrent renttype renttype_detail])
```

Comment:
can we also check the headers don't include `rent_type_detail` just for sanity's sake? Would make it clearer to anyone looking at this in the future that it's a previous column that's now been removed